mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -91,80 +61,89 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
     return data_thresholded, center_locations
 
 
-def gridding_2d(points, values, img_shape, method='linear', point_min=None,
-                point_max=None):
+def grided_inverse_fourier_transform_nd(points, values, grid, method):
     """
-    Interpolate non-Cartesian data into a cartesian grid
+    This function calculates the grided Inverse fourier transform
+    from Interpolated non-Cartesian data into a cartesian grid
 
     Parameters
     ----------
     points: np.ndarray
-        The 2D k_space locations of size [M, 2]
+        The N-D k_space locations of size [M, N]
     values: np.ndarray
-        An image of size [N_x, N_y]
-    img_shape: tuple
-        The final output ndarray
+        An image of size [N_x, N_y, N_z, ..]
```

Comment:
It's not an image, having image here is misleading

Reply:
Well, it is an image size itself. It represents what is the final image shape that we want, so that we organize the k-space in that format. This is more of target size of sensitivity maps and not the input size of kspace, so this makes more sense to me.